interface/patient_file/history/history.php

```diff
@@ -13,6 +13,8 @@
  require_once("history.inc.php");
  require_once("$srcdir/options.inc.php");
```

Comment:
Note I committed this code to the codebase here:
https://github.com/openemr/openemr/commit/fa10ca51f83d66e3240a23fc3b74b63390abbacd

I am going to point out a couple minor issues below(things I noted after committing above code :) ):